srcpkgs/android-udev-rules/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'android-udev-rules'
+pkgname=android-udev-rules
+version=20200613
+revision=1
+build_style=fetch
```

Comment:
- Remove `build_style`.

```sh
do_install() {
    vinstall 51-android.rules" 644 usr/lib/udev/rules.d
}
```

Commit message should be: `New package: android-udev-rules-20200613`